terraform/rtmp-server/nginx.service

```diff
@@ -0,0 +1,28 @@
+# Stop dance for nginx
```

Comment:
Is this a backport from later versions of nginx?

Reply:
I THINK it was taken from the service file in Raspbian GNU/Linux 8.
Previous versions of this server spawned nginx as a process durring startup, not a restartable service.